Casks/iina.rb

```diff
@@ -1,15 +1,20 @@
 cask "iina" do
-  version "1.2.0,129"
-  sha256 "91b87e80055f097a1cb7a8c91979deb5303315f2067552cbe7387f48bfc42736"
+  version "1.3.0"
+  sha256 "78959388f2a3f027c0187aa6702cba2d1019919ebca1f5bfba417f9b1d6528f8"
 
-  url "https://dl.iina.io/IINA.v#{version.csv.first}.dmg"
+  url "https://github.com/iina/iina/releases/download/v#{version}/IINA.v#{version}.dmg"
   name "IINA"
   desc "Free and open-source media player"
-  homepage "https://iina.io/"
+  homepage "https://github.com/iina/iina"
 
   livecheck do
-    url "https://www.iina.io/appcast.xml"
-    strategy :sparkle
+    url "https://github.com/iina/iina/releases/latest"
+    strategy :page_match do |page|
+      match = page.match(/(\d+(?:\.\d+)+)/i)
+      next if match.blank?
+
+      (match[1]).to_s
+    end
   end
 
```

Comment:
```suggestion
  livecheck do
    url :url
    strategy :github_latest
  end
```
This strategy should work here, and is preferential over `:page_match`